test/test_pipeline/implementations/test_util.py

```diff
@@ -5,11 +5,12 @@
 
 from autosklearn.pipeline.implementations.util import softmax
 
+
 class UtilTest(unittest.TestCase):
     def test_softmax_binary(self):
-        df = np.array([-40.00643897, 34.69754581, 23.71181359 -29.89724287,
+        df = np.array([-40.00643897, 34.69754581, 23.71181359 - 29.89724287,
```

Comment:
Is this supposed to really be a subtraction or there is a comma missing here?
(Same question for line 13)

Reply:
There is a comma missing

Reply:
OK